### DISCUSSION TITLE
Ubuntu building from source: missing dependencies

### DISCUSSION THREAD

**Discussion by Kikobeats** [libvips/libvips#3029] [Q&A]
Hello at

https://github.com/libvips/libvips/wiki/Build-for-Ubuntu#building-from-source

is listing dependencies, but `libvips` CI is using a different set of dependencies:

https://github.com/libvips/libvips/blob/master/.github/workflows/ci.yml#L54

Specifically, the following dependencies are missing in the wiki:

```
libexif-dev
libtiff5-dev
libcfitsio-dev
libpoppler-glib-dev
librsvg2-dev
libpango1.0-dev
libffi-dev
libopenjp2-7-dev
libimagequant-dev
libcgif-dev
```

Should they be added?

**Comment by jcupitt** [comment]
Hi @Kikobeats,

There have been problems with malicious libvipswiki edits, so we've had to restrict write access, unfortunately. This means it's no longer useful. 

We're planning to shut down the wiki and move any useful material in it into the main docs, but no one's found the time yet. There's an issue about this:

https://github.com/libvips/libvips/issues/2805

So in my opinion your time would be better spent making a PR to delete wiki pages.

**Reply by kmartinez** [reply]
I've added them to that wiki page for now - and am about to re-test a fresh Ubuntu to check.
Remind me where the new build-help docs are/will be?? I can port it over

**Reply by jcupitt** [reply]
Do you mean hints on using meson? The README has some notes:

https://github.com/libvips/libvips#building-from-source

**Reply by jcupitt** [reply]
tldr: for ubuntu, try 

```
meson build --prefix=/home/john/vips --libdir=lib --buildtype release
cd build
ninja
ninja install
```

It's a debug build by default, so you probably want to set `buildtype`. Debian puts libraries in `lib/lib-x86-linux/` by default, which is annoying, so `--libdir` turns that off.

**Reply by kmartinez** [reply]
Oh no meson is fine! I mean where to put a small set of "how-to" install on a couple of platforms - there is a lot in the README but it can be spelt out easier in a move from wiki page to docs. How about building-ubuntu.md building-win.md etc ?
However there is already https://www.libvips.org/install.html so maybe its redundant and fills up the repo with text docs

**Reply by jcupitt** [reply]
Ah gotcha. I think we should add chapters to the main docs and have everything in one place. We can get the "install" button on the website to just link to the doc chapter in the current libvips version.
